### ISSUE TITLE
Error while encrypting files: "The filename, directory name, or volume label syntax is incorrect.

### ISSUE THREAD

**Issue by rokshis** [rclone/rclone#964]
> What is your rclone version (eg output from `rclone -V`)
rclone v1.34

> Which OS you are using and how many bits (eg Windows 7, 64 bit)
Windows 7, 64 bit.

> Which cloud storage system are you using? (eg Google Drive)
Local drive. HDD. Just encrypting.

> The command you were trying to run (eg `rclone copy /tmp remote:tmp`)
rclone sync A:/directoryname localcrypt:directoryname

> A log from the command with the `-v` flag (eg output from `rclone -v copy /tmp remote:tmp`)
Error message:  "The filename, directory name, or volume label syntax is incorrect."

Full error: 
> " 2016/12/19 16:07:46 Ivairus/Soft/rclone-v1.34-windows-amd64/rclone-v1.34-windows-amd64/locally_crypt
ed/a4ujg8ujgfmr02bjc7k1uuqtuc/dasc3bku5k4mr1vb4odbck160g/f0v5kbg17e3p8o1fdffof8uhf0/bkkkqoldb0h6brcr
8emhib2bdg/k9pk49uahe7ss226l8dt39naauaidm1vhl8rbfpfn8sef226bonpskaerf172pbbjj0pror4d63jtj1ustu8iaql0
hksmpccdjcj1u8s86grimkqqbcrs560he84fo2nb5mbhqattg50rhmi2hi6cjckg8: Failed to copy: open \\?\A:\ROKO\
Ivairus\Soft\rclone-v1.34-windows-amd64\rclone-v1.34-windows-amd64\locally_crypted\a4ujg8ujgfmr02bjc
7k1uuqtuc\dasc3bku5k4mr1vb4odbck160g\5ftbcerumou67qrbeqkfq1ah24\lg560naf1456ns391k935j19fv58c4ph8hd4
5gag4tc5b76ajpqg\lg560naf1456ns391k935j19fv58c4ph8hd45gag4tc5b76ajpqg\k1otgi668suitabnjkoofijjls\mju
oc8db6q8bdnmbn9rv56unq05fg08tt6bnrfl4s704kp183e6g\raj6rcnf4uhqvmueagk71ltb9412qkv56nb7ru4qo014c4e0qh
4g\82psipedlis9ldtc40g9o9v1gc8oifkio9p0nk4diepjcm1h0urg\h0ug27mr0766ubj13him56ga3c5nmvbegi24sb22d8ev
d12v4flg\ceft2onfgsnuhlq1935tukpjmqa8j35la6teaanc70jvq8di7ac7pakpalkqi4oml7sjqigln18o2f29ahve8n3ac32
jn8cd3bc63i5u94kfk0d7ece5rjc6c3khf4l4t936qjgudm9v8kj7bejli3ge0eqr7d26l5rvh2obgkn6o7hpruv1qpg0svrcsib
f53h83tepcrttl9r58vbqau03ack6mu9np48628fsgv7liqukiv7fgo0kk0bqig0i: The filename, directory name, or
volume label syntax is incorrect.
"

I am trying to encrypt my 60GB folder with lots of huge and small files to a local folder. For most of them, everything passes by successfully, but there are few files that ocasionally get those errors, which means I might lose some crucial files.

**Comment by ncw** [comment]
Looks like you might have a file name that is very long.  Remember crypt expands file names by approx 60%...

I think the limit on file names might be 256 characters on windows - can you test and see if that is correct?  So the file name you are trying to crypt will be greater than 160 characters.

Does that make sense?

**Comment by rokshis** [comment]
I have solved the problem. It was my foolish mistake. I created an infinite loop, and new files that was being created was being copied over again and again.

But now, after realizing that, I fixed this issue and occurred with other one. (I fixed it, by moving the folder, where the crypted files where stored, out of the directory I was encrypting). 

Now the issue is that not all files are being synced. I get no errors, no warnings, but I can clearly see that, as my new folder size is 44.5GB and the original one is around 60GB. Or is it supposed to be this way? I mean, maybe there is some specific space saving algorithm or something.
